docker/test/integration/minifi/core/ImageStore.py

```diff
@@ -12,6 +12,7 @@ class ImageStore:
     def __init__(self):
         self.client = docker.from_env()
         self.images = dict()
+        self.test_dir = os.environ['PYTHONPATH'].split(':')[-1]  # Based on DockerVerify.sh
```

Comment:
This looks brittle, as someone may add another directory to the end of `PYTHONPATH` in the future.  It would be better if eg. DockerVerify.sh exported a dedicated `TEST_DIRECTORY` environment variable which we could read here.

EDIT: I can see this same code is already used in a few other places; I would change those, too.